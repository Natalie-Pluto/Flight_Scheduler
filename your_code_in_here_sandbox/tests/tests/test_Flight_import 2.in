flight import flights.csv
flight import flights6.csv
flight import
flight import what?
flights import flights.csv
flight
location import locations.csv
flight import flights6.csv
exit