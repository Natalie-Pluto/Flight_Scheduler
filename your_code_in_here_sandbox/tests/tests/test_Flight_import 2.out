User: Error reading file.

User: Imported 0 flights.
182 lines were invalid.

User: Error reading file.

User: Error reading file.

User: Flights
-------------------------------------------------------
ID   Departure   Arrival     Source --> Destination
-------------------------------------------------------
(None)

User: Usage:
FLIGHT <id> [BOOK/REMOVE/RESET] [num]
FLIGHT ADD <departure time> <from> <to> <capacity>
FLIGHT IMPORT/EXPORT <filename>

User: Imported 51 locations.

User: Imported 169 flights.
13 lines were invalid.

User: Application closed.
